### PR TITLE
Remove eventHandler

### PR DESCRIPTION
**Pull request by relu91**
As discussed #408 this PR removes references to `eventHandler` from the API.

***
<a href="https://pr-preview.s3.amazonaws.com/relu91/wot-scripting-api/pull/423.html" title="Last updated on Sep 26, 2022, 11:40 AM UTC (fc07cc7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-scripting-api/423/f7eaed5...relu91:fc07cc7.html" title="Last updated on Sep 26, 2022, 11:40 AM UTC (fc07cc7)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3398,6 +3322,12 @@ <h3>The <dfn>EventSubscriptionHandler</dfn> callback</h3>
                 Create an <a>Event</a> notification |response| according to the
                 <a>Protocol Bindings</a> from |data| and |subscriber|, including
                 its |options|.
+              </li>
+              <li>
+                  If |data| is `undefined`, assume that the notification |response| 
+                  will contain a `void` data payload.
```
Comment (zolkis) [inline index.html:3328]: I suggest reformulating to more exact prose, to something like

> If |data| is `undefined`, let |response|'s |data| be `undefined`

(needs to be done consistently over all occurrences of |response|

**Review by zolkis — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (danielpeintner) [reply index.html:3328]: Call of 2022-08-08
@relu91 will work (provide) an alternative to make clear what `void` means..

**Review by danielpeintner — APPROVED** [review]
LGTM after resolving https://github.com/w3c/wot-scripting-api/pull/423#discussion_r939993623

File: index.html [hunk 2]
```diff
@@ -3398,6 +3322,12 @@ <h3>The <dfn>EventSubscriptionHandler</dfn> callback</h3>
                 Create an <a>Event</a> notification |response| according to the
                 <a>Protocol Bindings</a> from |data| and |subscriber|, including
                 its |options|.
+              </li>
+              <li>
+                  If |data| is `undefined`, assume that the notification |response| 
+                  will contain a `empty` data payload as specified by <a>Protocol Bindings</a>.
```
Comment (zolkis) [inline index.html:3328]: This is too generic here. From an algorithm we need to point to exact steps or algorithms from another spec.
If the referred spec doesn't have that, we need to figure that out for the HTTP protocol binding and include the steps here. Let's track that in a separate issue.

Comment (danielpeintner) [inline index.html:3328]:
```suggestion
                  will contain an <emph>empty</emph> data payload as specified by <a>Protocol Bindings</a>. 
```

**Comment by danielpeintner** [comment]
Scripting Call 2022-09-26
Agreed to merge

File: index.html (hunk 2 above)
Comment (danielpeintner) [reply index.html]: see https://github.com/w3c/wot-scripting-api/issues/430

**Pull request merged**